### PR TITLE
401: Release Go CLI

### LINKED ISSUE #401: Automatically build CLI binaries on Travis
In order to make zally CLI usage easier for end customers, we need to build Zally binaries for the most popular platforms:

- OSX
- Linux
- ❓  Windows

### Things to check:

- Is it possible to do it automated way on Travis?
- Can we upload binaries to releases directly from Travis?

### Acceptance criteria

- Build script is created
- The binary files are checked on the platforms we are supporting
- ❓ Build is automated with Travis

Some inspiration: https://github.com/wallix/awless

### PR DESCRIPTION
**Pull request by netme**
Closes #401 

Contains release preparation instructions

**Proposal**: Let's keep release process manual for next release to feel it and then we can automate it using `goreleaser` tool. 

### How to test:
- Follow instructions in README.md from this PR

### CODE REVIEW

File: cli-go/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+zally/dist
```
Comment (netme) [inline cli-go/.gitignore]: Separate file will help us to move this project to another repo in the future

File: cli-go/README.md [hunk 2]
```diff
@@ -32,3 +32,44 @@ Not feature-complete, in active development
...
+1. Run `goreleaser`:
+
+    ```bash
+    cd cli-go/zally
+    goreleaser --skip-publish v1.1.0
+    ```
+
+1. Check builds inside `cli-go/zally/dist` directory.
+
+1. Publish release tag to GitHub:
```
Comment (maxim-tschumak) [inline cli-go/README.md]: Let's push the tag right after creating the tag. By this, we make sure, that the versions (tags) are somehow synchronized via the source code repository.

**Comment by maxim-tschumak** [comment]
And please add `dist/` to `.gitignore`

File: cli-go/README.md (hunk 2 above)
Comment (netme) [inline cli-go/README.md]: @maxim-tschumak that might be risky if something will go wrong with the build . What do you think about it?

Comment (maxim-tschumak) [inline cli-go/README.md]: Agree, another alternative would be to write pack all this in a script, which either fails and doesn't push/publish or does everything.
But let's keep it as-is for now - I'm OK with this :)

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by netme** [comment]
@maxim-tschumak I've rebased it, could you please re-approve it?

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**